src/__mocks__/attachments.ts

```diff
@@ -0,0 +1,71 @@
+/*!
```

Comment:
Maybe I'm missing something but I'm not sure how this file is used

Reply:
It was mock for local testing.... the attachements were empty...